docs/website/docs/command-reference/delete-component.md

```diff
@@ -36,11 +38,30 @@ If some resources attached to the component are present on the cluster, but not
 You can delete these resources by running the command in the [next section](#delete-without-access-to-devfile).
 :::
 
-By default, `odo` does not delete the Devfile, the `odo` configuration files, or the source code.
+#### Filtering resources to delete
+You can specify the type of resources candidate for deletion via the `--running-in` flag.
+Acceptable values are `dev` (for inner-loop resources) or `deploy` (for outer-loop resources).
+
+<details>
+<summary>Example</summary>
+
+```shell
+$ odo delete component --running-in dev
+Searching resources to delete, please wait...
+This will delete "my-nodejs" from the namespace "my-project".
+ •  The following resources will get deleted from cluster:
+ •      - Deployment: my-nodejs-app
+
+? Are you sure you want to delete "my-nodejs" and all its resources running in the Dev mode? Yes
+ ✓  Deleting resources from cluster [22ms]
+The component "my-nodejs" is successfully deleted from namespace "my-project"
+```
```

Comment:
Can you add this to `command-reference/docs-mdx/delete/<filename>.mdx` so that it can be later used for test automation?